userdocs/src/usage/experimental/gitops.md

```diff
@@ -197,25 +218,113 @@ the advantages of gitops moving forward.
 
 The basic command usage looks like this:
 
-```console
+```
 EKSCTL_EXPERIMENTAL=true eksctl enable profile --cluster <cluster-name> --region <region> --git-url=<url_to_your_repo> app-dev
 ```
 
 
 This command will clone the specified repository in your current working directory and then it will follow these steps:
 
-  1. add the component manifests of the Quick Start profile to your repository inside the `base/` folder
```

Comment:
These removed lines are the ones that should be there. This file is deleting the latest changes.